app/javascript/packages/polyfill/package.json

```diff
@@ -8,6 +8,7 @@
     "clipboard-polyfill": "^3.0.3",
     "custom-event-polyfill": "^1.0.7",
     "js-polyfills": "^0.1.43",
+    "promise-polyfill": "^8.2.3",
```

Comment:
Promises were only polyfilled previously because we use them in our own code, via [`@babel/preset-env`'s `useBuiltIns: 'usage'` option](https://github.com/18F/identity-idp/blob/93d572b48952596243ca57d0cf8c4b6dc5710ecc/babel.config.js#L36-L40). By default, third-party packages are [excluded](https://github.com/18F/identity-idp/blob/93d572b48952596243ca57d0cf8c4b6dc5710ecc/webpack.config.js#L55) from Babel transforms. Since some of the polyfills themselves rely on the `Promise` polyfill ([`fetch`](https://github.com/github/fetch#installation), [`webcrypto-shim`](https://github.com/vibornoff/webcrypto-shim#quick-start-with-bower)), it needs to be explicitly included.